device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/buffers_defaults_t0.j2

```diff
@@ -10,18 +10,22 @@
 {%- macro generate_buffer_pool_and_profiles() %}
     "BUFFER_POOL": {
         "ingress_lossless_pool": {
-            "size": "33329088",
+            {# SS Tab2 Row 71 #}
+            "size": "33555392",
             "type": "ingress",
             "mode": "dynamic",
-            "xoff": "7827456"
+            {# SS Tab2 Row 68 #}
+            "xoff": "8992256"
```

Comment:
For T0, the headroom pool needs to be calculated based on 40m uplinks and 5m downlinks. also 8 uplinks and remaining as downlinks. @baiwei0427 , comments?

Reply:
@neethajohn: can we update t0 later once we have usecase

Reply:
@abdosi  What cable assumptions do you hold when calculating buffer values?

Reply:
@baiwei0427 as i mention above change for t0 are same as t1 for now just to keep them synch. Idea of this PR is to update  for T1 only. I think for T0 we need to revisit once we have use-case

Reply:
@baiwei0427 , the assumptions for T1 are 16 uplinks 300m and 48 downlinks 40m.

Reply:
yes @neethajohn

Reply:
@abdosi , can we remove the numbers for T0 and update it later with the correct values. the pg_profile_lookup.ini is not going to change for T0

Reply:
@neethajohn updated to remove t0 buffer config and also reverted 100G 5m pg_profile value.